client/src/components/Profile/style.css

```diff
@@ -1,6 +1,11 @@
 .alignLeft {
     text-align: left;
-    color: white;
+    color: rgb(10, 10, 10);
```

Comment:
I think white text looks way better on profile but we can discuss that later before our final push

Reply:
done